assets/icons/wordpress-icons.js

```diff
@@ -17,3 +17,9 @@ export const checked = (
 		<Path d="M9 18.6L3.5 13l1-1L9 16.4l9.5-9.9 1 1z" />
 	</SVG>
 );
+
+export const button = (
```

Comment:
Why are we copying this here, rather than importing from `@wordpress/icons`?

Reply:
Not sure I just followed suit 😄 d26086a9a89fe8f7e9b44f94c6bba128fac5ecb1

Reply:
Sorry this was done for Wordpress compatibility — a dependency of the `@wordpress/icons` package (the `SVG` and similar components) was split off from `wp-components` into `wp-primitives` by core at some point, but that's not available in Wordpress 5.3. 
The alternative is hacking on the build config to maybe use those exports from `wp-components`, or polyfilling to `wp-primitives` file — it could be a bit of a timesink so this is the easier solution for now.

We could add a comment to the `wordpress-icons.js` file to indicate the reasoning.

Reply:
Cool removed the commit and added a comment so we should be ok now.

Reply:
Thanks, that's very helpful!